README.md

```diff
@@ -38,12 +38,12 @@ The flow is the main attraction of the application and has a lot of hidden featu
 Besides dragging and clicking the pipes, there are some additional options in the right-click menu:
 
 - Toggle activity mode: The pipes will change to a simpler and compacter appearance by clicking this option.
-- Realign flow: You can reset the flow to a generated flow.
+- Realign flow: You can regenerate the locations of the pipes (without changing the run-time behavior of your configuration).
```

Comment:
It not only moves the pipes, but also the listeners and exits, so maybe elements would be better.